test/unit/ConstantProductPair.t.sol

```diff
@@ -681,6 +682,7 @@ contract ConstantProductPairTest is BaseTest, IReservoirCallee {
 
     function testPlatformFee_DisableReenable() external {
         // sanity
+        assertGt(_constantProductPair.platformFee(), 100);
```

Comment:
why specifically asserting that it is greater than 100?

Reply:
That it's non-zero, could change to >0 if we want. Did 100 to avoid chances of roudning platform mint to zero, though I think it wont even if platformFee is 1